lib/lighthouse/letters_generator/configuration.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/response/raise_error'
+require 'lighthouse/auth/client_credentials/jwt_generator'
+require 'lighthouse/auth/client_credentials/service'
+
+module Lighthouse
+  module LettersGenerator
+    class Configuration < Common::Client::Configuration::REST
+      SETTINGS = Settings.lighthouse.letters_generator
+
+      def path_join(*paths)
+        paths.reduce('') do |acc, p|
+          trimmed_slash = p.gsub(%r{(^/+|/+$)}, '')
+          acc + "#{trimmed_slash}/"
+        end.chop!
+      end
+
+      def generator_url
+        URI path_join(SETTINGS.url, SETTINGS.path)
+      end
+
+      def service_name
+        'Lighthouse_LettersGenerator'
+      end
+
+      def connection
+        @conn ||= Faraday.new(generator_url, headers: base_request_headers, request: request_options) do |faraday|
+          faraday.use      :breakers
+          faraday.use      Faraday::Response::RaiseError
+
+          faraday.request :multipart
```

Comment:
Wondering if this is needed? I realize that I'm using this is my configuration as well, which is part of the reason I am asking because I imagine this is mostly needed if doing `POST` requests